dbterd/adapters/targets/mermaid/mermaid_test_relationship.py

```diff
@@ -14,6 +17,29 @@ def run(manifest, catalog, **kwargs):
     return ("output.md", parse(manifest, catalog, **kwargs))
 
 
+def replace_column_name(column_name: str) -> str:
+    return column_name.replace(" ", "-").replace(".", "__")
+
+
+def match_complex_column_type(column_type: str) -> Optional[str]:
+    pattern = r"(\w+)<(\w+\s+\w+(\s*,\s*\w+\s+\w+)*)>"
+    match = re.match(pattern, column_type)
+    if match:
+        return match.group(1)
+    else:
+        return None
+
+
+def replace_column_type(column_type: str) -> str:
```

Comment:
There are many possible ways to abbreviate or omit this character, and it may be possible to change it from an environment variable or command line argument. However, we want to start out with a simple form.